gcloud/logging/test__gax.py

```diff
@@ -0,0 +1,457 @@
+        self.assertEqual(entry.log_name, '')
+        self.assertEqual(entry.resource.type, '')
+        self.assertEqual(entry.labels, {})
+        self.assertEqual(entry.text_payload, TEXT)
+        self.assertEqual(entry.severity, WARNING)
+
+        entry = entries[1]
+        self.assertTrue(isinstance(entry, LogEntry))
+        self.assertEqual(entry.log_name, '')
+        self.assertEqual(entry.resource.type, '')
+        self.assertEqual(entry.labels, {})
+        json_struct = entry.json_payload
+        self.assertTrue(isinstance(json_struct, Struct))
+        self.assertEqual(json_struct.fields['payload'].string_value,
+                         JSON['payload'])
+        operation = entry.operation
+        self.assertEqual(operation.producer, PRODUCER)
+        self.assertEqual(operation.id, OPID)
+
+        entry = entries[2]
+        self.assertTrue(isinstance(entry, LogEntry))
+        self.assertEqual(entry.log_name, '')
+        self.assertEqual(entry.resource.type, '')
+        self.assertEqual(entry.labels, {})
+        proto = entry.proto_payload
+        self.assertTrue(isinstance(proto, Any))
+        self.assertEqual(proto.type_url, TIMESTAMP_TYPE_URL)
+        request = entry.http_request
+        self.assertEqual(request.request_url, URL)
+
+        self.assertEqual(log_name, LOG_PATH)
+        self.assertEqual(resource, RESOURCE)
+        self.assertEqual(labels, LABELS)
+        self.assertEqual(partial_success, False)
+        self.assertEqual(options, None)
+        # pylint: enable=too-many-statements
+
+    def test_logger_delete(self):
+        LOG_PATH = 'projects/%s/logs/%s' % (self.PROJECT, self.LOG_NAME)
+        gax_api = _GAXLoggingAPI()
+        api = self._makeOne(gax_api)
+
+        api.logger_delete(self.PROJECT, self.LOG_NAME)
+
+        log_name, options = gax_api._delete_log_called_with
+        self.assertEqual(log_name, LOG_PATH)
+        self.assertEqual(options, None)
+
+
+class _GAXLoggingAPI(object):
+
+    def __init__(self, **kw):
+        self.__dict__.update(kw)
+
+    def list_log_entries(
+            self, projects, filter_, order_by, page_size, options):
+        self._list_log_entries_called_with = (
+            projects, filter_, order_by, page_size, options)
+        return self._list_log_entries_response
+
+    def write_log_entries(self, entries, log_name, resource, labels,
+                          partial_success, options):
+        self._write_log_entries_called_with = (
+            entries, log_name, resource, labels, partial_success, options)
+
+    def delete_log(self, log_name, options):
+        self._delete_log_called_with = log_name, options
+
+
+class _HTTPRequestPB(object):
+
+    request_url = 'http://example.com/requested'
+    request_method = 'GET'
+    status = 200
+    referer = 'http://example.com/referer'
+    user_agent = 'AGENT'
+    cache_hit = False
+    request_size = 256
+    response_size = 1024
+    remote_ip = '192.168.1.254'
```

Comment:
I know this is a local IP but any reason behind using this one in particular? I see you used 1.2.3.4 earlier. Not sure if this could ever route to something in the build environments.